bin/merge

```diff
@@ -90,3 +97,59 @@ else
     echo "error: ${repo}#${pull_request} has already been merged"
     exit 1
 fi
+
+}
+
+usage() {
+
+cat << EOF
+Usage: $0 [-h] [-r "REPO-NAME PR-NUMBER"]
```

Comment:
You can have non-flagged arguments with getopts, you just have to handle them after the getopts block.